### DISCUSSION TITLE
Ebus Adapter v5: no signal

### DISCUSSION THREAD

**Discussion by hemagi** [john30/ebusd#1412] [Q&A]
Hi,

I have been using the ebus adapter v3 for a number of years now. Additionally I ordered an adapter as backup and got a v5.

Today I tried to get it running in parallel to the v3 adapter, without success.

After plugging in the v5 adapter via USB in a proxmox lxc (comparable to the v3 adatper) the blue LED is blinking.
ebusctl info tells me that the device itself is accessible:
```
version: ebusd 24.1.24.1
update check: OK
device: /dev/ttyEbusAdapter, serial high speed, enhanced
access: *
signal: no signal
reconnects: 0
masters: 1
messages: 17
conditional: 0
poll: 0
update: 10
address 31: master #8, ebusd
address 36: slave #8, ebusd
```

After plugging in the ebus cables parallel to the v3 adapter the blue and green leds of the v5 adapter are on permanently. The red one is off.
Nothing is blinking. ebusctl i continues to report `no signal`.

The v3 adapter in parallel continues to work w/o issues.

I tried the enh: protocol. No change. A changed the bus cables. No change.

Any help is appreciated

**Comment by yvan0267** [comment]
Hello, strange, on the device line you have no indication of the firmware version of the device.
Maybe try a firmware update.

**Comment by hemagi** [comment]
In between I found a solution. I had to log in via WiFi. There I changed the access method to USB, which was not set before. I was then able to access the adapter via USB.
Then ebusctl told me that a newer firmware was available. So I updated it.

However, it is worth noting that now neither the currently installed firmware nor the availability of a newer firmware is shown by ebusctl. Is this intended? How can I get this information?
